x-pack/plugin/sql/jdbc/src/test/java/org/elasticsearch/xpack/sql/jdbc/jdbc/JdbcPreparedStatementTests.java

```diff
@@ -129,7 +129,7 @@ public void testThrownExceptionsWhenSettingByteTypeValues() throws SQLException
         JdbcPreparedStatement jps = createJdbcPreparedStatement();
 
         SQLException sqle = expectThrows(SQLException.class, () -> jps.setObject(1, (byte) 6, Types.TIMESTAMP));
-        assertEquals("Unable to convert value [6] of type [TINYINT] to a Timestamp", sqle.getMessage());
+        assertEquals("Unable to convert value [6] of type [BYTE] to [TIMESTAMP]", sqle.getMessage());
```

Comment:
Which is what `byte` maps to. The param sent to the driver is a number which we map to our own type. If the user would have sent a `JDBCType.BYTE/TIMESTAMP` it would be correctly reported in the message.